### ISSUE TITLE
[Bug] All generated content disappear when finished

### ISSUE THREAD

**Issue by plmsmile** [alphaXiv/feedback#135]
When I select some content and chat with the assistant,  the all llm generated content suddenly disappeared when LLM generation is finished.

**Asked**

<img width="1096" height="1380" alt="Image" src="https://github.com/user-attachments/assets/f672a6d7-525f-41ec-8402-6c22bffedfa7" />

**But Nothing there.**  **Had Tried Many Times.** **Very bad experience**

<img width="1144" height="1530" alt="Image" src="https://github.com/user-attachments/assets/11b8b898-a0b2-4c3e-bdbc-ffdb4c88d937" />

**Comment by sox8502** [comment]
@plmsmile I'm sorry about the bad experience, can you share what paper you were chatting with?

**Comment by plmsmile** [comment]
> [@plmsmile](https://github.com/plmsmile) I'm sorry about the bad experience, can you share what paper you were chatting with?

Thanks for your reply. 
## 1. Current Bug (generation disappear Bug)
In fact it occurs in many papers (recent months). 

What I read yesterday was the **DeepSeek-V3.2**。https://www.alphaxiv.org/abs/2512.02556

What I selected is **part 2.1.1 Continue Pretraining**. And the prompt is "**翻译和讲解一下**"。

> Dense Warm-up Stage. We first use a short warm-up stage to initialize the lightning indexer.
In this stage, we keep dense attention and freeze all model parameters except for the lightning
indexer. To align the indexer outputs with the main attention distribution, for the 𝑡-th query
token, we first aggregate the main attention scores by summing across all attention heads.
This sum is then L1-normalized along the sequence dimension to produce a target distribution
𝑝𝑡,: ∈ R𝑡 . Based on 𝑝𝑡,:, we set a KL-divergence loss as the training objective of the indexer:
L𝐼 =∑︁
𝑡
DKL
𝑝𝑡,: Softmax𝐼𝑡,:
. (3)
For warm-up, we use a learning rate of 10−3. We train the indexer for only 1000 steps, with each
step consisting of 16 sequences of 128K tokens, resulting in a total of 2.1B tokens.
Sparse Training Stage. Following indexer warm-up, we introduce the fine-grained token
selection mechanism and optimize all model parameters to adapt the model to the sparse
pattern of DSA. In this stage, we also keep aligning the indexer outputs to the main attention
distribution, but considering only the selected token set S𝑡 = 𝑠 𝐼𝑡,𝑠 ∈ Top-k𝐼𝑡,:
 :
L𝐼 =∑︁
𝑡
DKL
𝑝𝑡,S𝑡 Softmax𝐼𝑡,S𝑡
. (4)
It is worth noting that we detach the indexer input from the computational graph for separate
optimization. The training signal of the indexer is from only L𝐼 , while the optimization of the
main model is according to only the language modeling loss. In this sparse training stage, we
use a learning rate of 7.3 × 10−6, and select 2048 key-value tokens for each query token. We train
both the main model and the indexer for 15000 steps, with each step consisting of 480 sequences
of 128K tokens, resulting in a total of 943.7B tokens.


Thanks for the creation of the great tool alphaXiv. I use it everyday and it's my most important paper tool. 

## 2. Another 2 important Issues in my view. 
However, there are 2 important experience issues in my view which confused me a long time. And I wanna search help from you.

### **2.1 Select Chat related Bug . (Very Important!!)** 

When **I select a long context** (eg. 2paragraphs or a long paragraph), the alphaxiv will not give **all my select content** to the llm assistant.  And the llm will **return a very bad response** to me, because **it did not see all my select content.**

See more detail in this issue：https://github.com/alphaXiv/feedback/issues/112


### 2.2 Translation Monica-AI did not work in alphaxiv (Translation related).

- I often need to **translate(explain) some sentences of paper** (eg. some paragraphs). 
-  I use a useful tool [monica-ai chrome plugin](https://chromewebstore.google.com/detail/monica-all-in-one-ai-assi/ofpnmcalabcbjgholdjcjblkibolbppb?pli=1) **in other webs**.

- Like this image:  After select the content, I only need to click the translation button and then it will show the translation result. Very Useful.
<img width="1584" height="222" alt="Image" src="https://github.com/user-attachments/assets/dfd9825f-5174-4734-be25-1589e754ba43" />

<img width="1396" height="726" alt="Image" src="https://github.com/user-attachments/assets/ac8a3b91-37bc-4359-8998-d16cce6eb9be" />

- **However, It can not work in alphaxiv. After I selected the content, the monica-ai tool will not appear.**

<img width="1742" height="568" alt="Image" src="https://github.com/user-attachments/assets/3822fe23-eaac-4a0d-9a67-586fc8b8699c" />

- **However, It only work when click a single word, too restricted.**

<img width="442" height="154" alt="Image" src="https://github.com/user-attachments/assets/67f8f7e4-7128-42bf-ab91-3d714765b716" />




Thanks for the reply and hope your better alphaxiv. Thanks again.
